src/js/edu-benefits/utils/validations.js

```diff
@@ -284,8 +288,33 @@ function isValidDirectDepositPage(data) {
   return isValidField(isValidRoutingNumber, data.bankAccount.routingNumber);
 }
 
+function isValidBenefitsHistoryPage(data) {
+  return data.activeDutyRepaying.value !== 'Y' ||
+    (!isBlankDateField(data.activeDutyRepayingPeriod.from)
+    && !isBlankDateField(data.activeDutyRepayingPeriod.to)
+    && isValidDateRange(data.activeDutyRepayingPeriod.from, data.activeDutyRepayingPeriod.to));
+}
+
+function isValidRotcScholarshipAmount(data) {
+  return isValidField(isValidMonetaryValue, data.amount) && isValidField(isValidYear, data.year);
+}
+
+function isValidRotcHistoryPage(data) {
+  return data.seniorRotc.rotcScholarshipAmounts.every(isValidRotcScholarshipAmount);
+}
+
 function isValidForm(data) {
-  return isValidBenefitsInformationPage(data);
+  return isValidBenefitsInformationPage(data)
```

Comment:
I'm wary of the amount of repetition happening in validations.js-- Given that each field has validation rules attached to it, It seems like we could iterate through all of them?

On the other hand, this does make it easier to introduce new higher-level validation rules, which I could see us doing...